### DISCUSSION TITLE
unable to open SQL Anywhere DB with jTDS driver

### DISCUSSION THREAD

**Discussion by Henry-Jones** [dbeaver/dbeaver#11689] [Q&A] [answered]
Hello,

If I try to open a SQL Anywhere Database 12 with the jTDS driver the following error occurs:

the top error message means "the database was not found"
the message in the details then states "login failed"

If I open the exact same database with the jConnect driver it works.

What could be the problem here ?

I had opened this issue before but it got tangled with another issue on the same thread so it was abandoned there.

#### System information: 
- OS: Windows 10 20H2
- DBeaver: 7.3.4
- Extensions: none

#### Connection specification:
- Windows Server 2012 R2 x64
- Database: SQL Anywhere 12
- jTDS (Version Info not found, but much likely the latest)
- Do you use tunnels or proxies (SSH, SOCKS, etc): no

#### Describe the problem you're observing:
see above

#### Steps to reproduce, if exist:
try to open a SQL Anywhere 12 Database with the jTDS driver

#### Include any warning/errors/backtraces from the logs
!ENTRY org.jkiss.dbeaver.model 4 0 2021-02-26 12:38:09.559
!MESSAGE SQL Anywhere-Fehler -83: Angegebene Datenbank nicht gefunden

![image](https://user-images.githubusercontent.com/9108755/109296488-a65fed80-7830-11eb-970e-ddc4995397b7.png)

**Comment by uslss** [comment, marked as answer]
Seems it is expected behavior. jtds driver is needed for old database versions.
